### ISSUE TITLE
Getting health of the ClamAV anti-virus when it's disabled gives a server error.

### ISSUE THREAD

**Issue by alexweininger** [appwrite/appwrite#1171]
## To Reproduce

1. Disable ClamAV
2. call `health.getAntiVirus()` in the `node-appwrite` sdk. (only confirmed here so far, but probably is an error not just in sdk)

(Write your steps here:)

## Expected behavior

Should return something resembling "ClamAV disabled"

(Write what you thought would happen.)

## Actual Behavior
```js
{
  message: "Server Error",
  code: 500,
  version: "0.8.0",
}
```

(Write what happened. Add screenshots, if applicable.)

## Your Environment

(Write Environment, Operating system and version etc.)

ClamAV is disabled running with Appwrite 0.8.

**Comment by lohanidamodar** [comment]
@alexweininger We believe the issue was caused by something else? When Antivirus is properly disabled, we get correct health status. If I'm wrong and if the error still persist please, feel free to reopen the issue and provide more information. Thank you.

**Comment by lohanidamodar** [comment]
@alexweininger Actually we found the problem, the bug is when ClamAV is actually enabled but the ClamAV server is not running, this PR #1188 should fix this issue.

**Comment by eldadfux** [comment]
This is now fixed and available as part of Appwrite 0.9:
https://dev.to/appwrite/announcing-appwrite-0-9-the-open-source-firebase-alternative-53ho 🥳 🎉
